di_website/templates/includes/partials/side_nav.html

```diff
@@ -0,0 +1,22 @@
+<aside class="l-sidebar__aside">
```

Comment:
Definitely shouldn't hardcode anything in this file ... the `secondary_menu` tag shall handle it... you just need to set it's parent to `self.get_parent`, and as long as these pages are tied to the correct parent (Who we are), they'll show the correct menu, even highlighting the current page.

Reply:
That came in the rebase, working on it